notebook/gen_ai_jss.ipynb

```diff
@@ -0,0 +1,818 @@
+      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /opt/conda/lib/python3.7/site-packages (from requests<3.0.0dev,>=2.21.0->google-cloud-bigquery->-r requirements.txt (line 3)) (1.26.14)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /opt/conda/lib/python3.7/site-packages (from requests<3.0.0dev,>=2.21.0->google-cloud-bigquery->-r requirements.txt (line 3)) (3.4)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /opt/conda/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r requirements.txt (line 5)) (0.4.8)\n",
+      "Building wheels for collected packages: grapheme\n",
+      "  Building wheel for grapheme (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for grapheme: filename=grapheme-0.6.0-py3-none-any.whl size=210080 sha256=151ed5af1b31e4778f090142fa4a8371b26835db19fc17e1d45eb6cb3a227c55\n",
+      "  Stored in directory: /var/tmp/pip-ephem-wheel-cache-fsg3gj6m/wheels/30/52/c4/1eefeda7e11cf06fb2f7ee3cde4049b0e915725f3bb9b5be45\n",
+      "Successfully built grapheme\n",
+      "Installing collected packages: polling2, grapheme, about-time, alive-progress\n",
+      "Successfully installed about-time-4.2.1 alive-progress-3.1.4 grapheme-0.6.0 polling2-0.5.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install the packages\n",
+    "import os\n",
+    "\n",
+    "if not os.getenv(\"IS_TESTING\"):\n",
+    "    USER = \"--user\"\n",
+    "else:\n",
+    "    USER = \"\"\n",
+    "! pip3 install {USER} --upgrade -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "58707a750154"
+   },
+   "source": [
+    "### Colab only: Uncomment the following cell to restart the kernel."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "f200f10a1da3"
+   },
+   "outputs": [],
+   "source": [
+    "# Automatically restart kernel after installs so that your environment can access the new packages\n",
+    "# import IPython\n",
+    "\n",
+    "# app = IPython.Application.instance()\n",
+    "# app.kernel.do_shutdown(True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BF1j6f9HApxa"
+   },
+   "source": [
+    "## Before you begin\n",
+    "\n",
+    "### Set up your Google Cloud project\n",
+    "\n",
+    "This notebook assumes that you have already deployed this solution using either the [Terraform script]() **TODO: fix target for link** or using the [Solutions console](https://console.cloud.google.com/products/solutions/catalog). During this deployment, several actions required to run this solution were performed on your behalf:\n",
+    "\n",
+    "1. The [Cloud Function](https://console.cloud.google.com/functions/list) was deployed.\n",
```

Comment:
I would prefer to remove the 1., 2., 3., 4. to make it more aesthetically appealing.

Reply:
Done.